app/views/reminder_mailer/_sign_off_msg.html.erb

```diff
@@ -1,4 +1,4 @@
-<p>Are you new to sending pull requests? Check out the guides here: <%= contributing_url %></p>
-<p>You can also follow @24pullrequests on twitter: <%= twitter_url %></p>
-<p>You can change frequency of these emails here: <%= preferences_url %></p>
-<p>You can unsubscribe to these emails here: <%= user_unsubscribe_url(@user) %></p>
+<p>Are you new to sending pull requests? Check out the guides here: <%= link_to contributing_url, contributing_url %></p>
+<p>You can also follow @24pullrequests on twitter: <%= link_to twitter_url, twitter_url %></p>
+<p>You can change frequency of these emails here: <%= link_to preferences_url, preferences_url %></p>
+<p>You can unsubscribe to these emails here: <%= link_to user_unsubscribe_url(@user.unsubscribe_token), user_unsubscribe_url(@user.unsubscribe_token) %></p>
```

Comment:
If you need to call method twice like that, just assign it to variable first:
```erb
<%- url = user_unsubscribe_url(@user.unsubscribe_token) -%>
<%= link_to url, url %>
```

But in this case it enough to:
```erb
<%= link_to nil, user_unsubscribe_url(@user.unsubscribe_token) %>
```